api/controllers/game/stalemate-reject.js

```diff
@@ -5,24 +5,17 @@ module.exports = async function (req, res) {
   try {
     const game = await Game.findOne({ id: req.session.game }).populate('players');
 
-    const gameUpdates = {
-      turnStalemateWasRequestedByP0: -1,
-      turnStalemateWasRequestedByP1: -1,
-    };
```

Comment:
Removed this to disallow making multiple stalemate requests in one turn. That's mainly because in the GameState API, each stalemate request writes a row to the `GameStateRow` table, so we don't want to let you keep cluttering up the table with a bajillion stalemate requests. That isn't a problem for the legacy API, since it only records the 'latest' stalemate requests, but adding this constraint ensures that the legacy API is consistent with the GameStateAPI.

So this block + the `Game.update()` call below would effectively 'reset' the last time a user requested a stalemate to allow them to continuously request them back and forth. Now the requirement is you can only request a stalemate once per turn so I'm removing the reset.